db/migrate/20220224202612_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] While creating the comment, we need the user or `author_id and post_id` to be included in the comment table and a corresponding index